Makefile

```diff
@@ -69,8 +70,7 @@ help:
 	@echo "- format             Format the python source code"
 	@echo "- lint               Lint the python source code"
 	@echo "- format-lint        Format and lint the python source code"
-	@echo "- test               Run the tests"
-	@echo -e " \033[1mLOCAL SERVER TARGETS\033[0m "
+	@echo "- test               Run the tests"$(YAPF_CMD) -p -i --style .style.yapf $(PYTHON_FILES)
 	@echo "- serve              Run the project using the flask debug server"
```

Comment:
Oops middle mouse button paste error I guess ;-)

Reply:
Oops, Sorry for that. I hope know there are no more paste error ;-)